libs/net/kafka/Models/NotificationDestination.cs

```diff
@@ -0,0 +1,21 @@
+namespace TNO.Kafka.Models;
+
+/// <summary>
+/// NotificationDestination enum, provides a way to control where to send a notification.
+/// </summary>
+[Flags]
```

Comment:
This will allow us to send a single notification request to multiple destinations.